test/1.4/legacy/T_shared_logs_on_device_disabled.dml

```diff
@@ -0,0 +1,44 @@
+/*
+  © 2023 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+
+device test;
+
+/// DMLC-FLAG --simics-api=6
+/// DMLC-FLAG --no-compat=shared_logs_on_device
+
+template t is _qname {
+    shared method hello() {
+        log info: "hello from %s", _qname();
+    }
+}
+
+is t;
+group g is t {
+    in each object { is t; }
+
+    group c[i < 2];
+
+    subdevice s[i < 3] {
+        group c[j < 5][k < 7] is t {
+            subdevice s;
+        }
+    }
+}
+
+method init() {
+    /// GREP \[obj info\] hello from test
+    hello();
+    /// GREP \[obj info\] hello from g
+    g.hello();
+    /// GREP \[obj info\] hello from g.c\[1\]
+    g.c[1].hello();
+    /// GREP \[obj\.g\.s\[2\] info\] hello from g\.s\[2\]
+    g.s[2].hello();
+    /// GREP \[obj\.g\.s\[2\] info\] hello from g\.s\[2\]\.c\[3\]\[5\]
+    g.s[2].c[3][5].hello();
+    /// GREP \[obj\.g\.s\[2\]\.c\[3\]\[5\]\.s info\] hello from g\.s\[2\]\.c\[3\]\[5\]\.s
```

Comment:
wonderful escapes

Reply:
Hah! Yes. You think we should add an option to `GREP` so it doesn't operate on a regex?

Reply:
no it's fine! simple is good. I just couldn't resist admiring the beauty.